src/src/Helpers/SystemVariables.cpp

```diff
@@ -73,8 +73,7 @@ void SystemVariables::parseSystemVariables(String& s, boolean useURLencode)
       case BSSID:             value = String((wifiStatus == ESPEASY_WIFI_DISCONNECTED) ? F("00:00:00:00:00:00") : WiFi.BSSIDstr()); break;
       case CR:                value = "\r"; break;
       case IP:                value = getValue(LabelType::IP_ADDRESS); break;
-      case IP4:               value = WiFi.localIP().toString().substring(WiFi.localIP().toString().lastIndexOf('.') + 1); break; // 4th IP
-                                                                                                                                  // octet
+      case IP4:               value.concat( (int) WiFi.localIP()[3] ); break; // 4th IP octet
```

Comment:
Just curious, why use `concat` here?

Reply:
were not sure if += will add numer or char, but when cast to int it will use proper method
but yes it can be also like: 
`value = String( (int) WiFi.localIP()[3] ));` can update it if you want

Reply:
Better have it do an assign instead of a concat.
You never know what's in the `value` when you start calling it.

Reply:
looks like concat wil not double copy string

Reply:
concat sounds like `+=`

Reply:
yes but it not create another string to copy from like assign does and write directly to object

Reply:
https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/WString.cpp#L362-L366
https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/WString.cpp#L325-L341

Reply:
Like `emplace_back` used in some other container types.

Reply:
simmilar :)

ok so is it ready to merge?